project/Dependencies.scala

```diff
@@ -62,7 +62,7 @@ object Dependencies {
   val jodaTime: ModuleID =        "joda-time"                     % "joda-time"             % "2.10.14"
   val jodaConvert: ModuleID =     "org.joda"                      % "joda-convert"          % "2.2.2"
   val typesafeConfig: ModuleID =  "com.typesafe"                  % "config"                % "1.4.2"
-  val sentryLogback: ModuleID =   "io.sentry"                     % "sentry-logback"        % "1.7.30"
+  val sentryLogback: ModuleID =   "io.sentry"                     % "sentry-logback"        % "6.2.1"
```

Comment:
Can we upgrade even further to 6.3? Sentry is suggesting it in the alert you linked in the description
![Screen Shot 2022-07-20 at 10 21 20 AM](https://user-images.githubusercontent.com/19961550/180006314-4c50e5d8-8e96-4410-b017-19c8a0565586.png)